bundles/persistence/org.openhab.persistence.jdbc/java/org/openhab/persistence/jdbc/db/JdbcBaseDAO.java

```diff
@@ -77,6 +87,42 @@ public JdbcBaseDAO() {
         initSqlQueries();
     }
 
+    /**
+     * ## Get high precision by fractal seconds, examples ##
+     *
+     * mysql > 5.5 + mariadb > 5.2:
+     * DROP TABLE FractionalSeconds;
+     * CREATE TABLE FractionalSeconds (time TIMESTAMP(3), value TIMESTAMP(3));
+     * INSERT INTO FractionalSeconds (time, value) VALUES( NOW(3), '1999-01-09 20:11:11.126' );
```

Comment:
The code checks for >5.5, but MySQL 5.6.2 and 5.6.3 do not support fractional seconds.  The documentation says the fsp arguments are ignored in those releases, hence my suggestion to document how the code in this PR does not precisely match MySQL release behavior.